docs/src/Module.doc.js

```diff
@@ -46,7 +46,21 @@ card(
         name: 'iconAccessibilityLabel',
         href: 'static-icon',
         type: 'string',
-        description: 'Label to provide information about the icon used for screen readers',
+        description:
+          'Label to provide information about the icon used for screen readers. Be sure to localize the label.',
```

Comment:
These aren't new, just added notes about localization (and reworded the id description)